Frontend/ui-customer/src/Screens/CustomerUtils/MyRidesScreen/Controller.purs

```diff
@@ -242,6 +247,9 @@ getFares fares = {
 , pickupCharges : "₹ 10.0"
 , waitingCharges : "₹ " <> show (getFareFromArray fares WAITING_CHARGES)
 , nominalFare : "₹ " <> show (getFareFromArray fares DRIVER_SELECTED_FARE)
+, referenceString : ("1.5" <> (getString DAYTIME_CHARGES_APPLICABLE_AT_NIGHT)) <> "\n\n" <>
+                        (getString DRIVERS_CAN_CHARGE_AN_ADDITIONAL_FARE_UPTO) <> "\n\n" <>
```

Comment:
We should send the also reference if we don't have driver offered fares no need of showing this line. same for waiting charges